src/pkg/packager/common.go

```diff
@@ -249,6 +253,8 @@ func (p *Packager) loadZarfPkg() error {
 		return fmt.Errorf("unable to handle the provided package path: %w", err)
 	}
 
+	wasPulled := p.cfg.DeployOpts.PackagePath == p.tmp.Base
```

Comment:
```suggestion
	extractedToTmp := p.cfg.DeployOpts.PackagePath == p.tmp.Base
```

I think this could have a better name to explain why you are doing this.